webhook/config_validation_controller.go

```diff
@@ -38,17 +38,19 @@ import (
 
 // ConfigValidationController implements the AdmissionController for ConfigMaps
 type ConfigValidationController struct {
+	name         string
```

Comment:
Can we comment on what `name` and especially `path` mean here?

Reply:
Done